recipe/meta.yaml

```diff
@@ -32,4 +32,4 @@ about:
 
 extra:
   recipe-maintainers:
-    -janschulz
\ No newline at end of file
+    - janschulz
```

Comment:
> But now I'm janschulz:-)

I like inconsistency - there will be others who have non lowercased names which forge needs to handle. If you wanted to keep it capitalised, you should! :+1: